.eslintrc.js

```diff
@@ -1,45 +1,132 @@
 module.exports = {
+  'root': true,
   'extends': [
-    'eslint:recommended',
-    'plugin:json/recommended'
+    'plugin:@okta/okta/courage-app' // apply courage-app rules to all files including properties
   ],
-  'env': {
-    'browser': true,
-    'jasmine': true,
-    'node': true,
-    'amd': true
-  },
   'parser': 'babel-eslint',
   'parserOptions': {
     'sourceType': 'module',
     'ecmaVersion': 2017
   },
-  'plugins': [
-    'eslint-plugin-local-rules',
-    'eslint-plugin-no-only-tests'
-  ],
-  'globals': {
-    'spyOnEvent': false,
-    'JSON': true,
-    'DEBUG': true,
-    'Promise': true
-  },
   'overrides': [
     {
-      'files': ['src/**/*.js'],
+      // temporarily ignoring files that violate the @okta/okta/no-exclusive-language rule
+      'files': [
+        'packages/**/login.properties',
+        'packages/**/country_fr.properties',
+        'buildtools/commands/generate-phone-codes.js',
+        'stylelint.config.js',
+        'webpack.playground.config.js'
+      ],
       'rules': {
-        'local-rules/no-bare-templates': 2,
-      }
+        '@okta/okta/no-exclusive-language': 0,
+      },
     },
     {
-      'files': ['packages/**/*.json'],
+      'files': ['*.js'], // apply eslint:recommended rules to all js files
+      'extends': [
+        'eslint:recommended',
+      ],
+      'env': {
+        'browser': true,
+        'jasmine': true,
+        'node': true,
+        'amd': true
+      },
+      'plugins': [
+        'eslint-plugin-local-rules',
+        'eslint-plugin-no-only-tests'
+      ],
+      'globals': {
+        'spyOnEvent': false,
+        'JSON': true,
+        'DEBUG': true,
+        'Promise': true
+      },
       'rules': {
-        'local-rules/no-missing-keys': 2,
-      }
+        'camelcase': 2,
+        'complexity': [2, 10],
+        'curly': 2,
+        'eqeqeq': 2,
+        'guard-for-in': 2,
+        'indent': [2, 2, { 'VariableDeclarator': 2 }],
+        'max-depth': [2, 2],
+        'max-len': [2, 120],
+        'max-params': [2, 10],
+        'max-statements': [2, 20],
+        'new-cap': [2, { 'properties': false, 'capIsNewExceptions': ['Q'] }],
+        'no-caller': 2,
+        'no-eval': 2,
+        'no-implied-eval': 2,
+        'no-new': 2,
+        'no-unused-expressions': [2, { 'allowShortCircuit': true, 'allowTernary': true }],
+        'no-use-before-define': [2, 'nofunc'],
+        'no-only-tests/no-only-tests': 'error',
+        'quotes': [2, 'single'],
+        'semi': 2,
+        'space-before-function-paren': [2, {
+          'anonymous': 'never',
+          'named': 'never',
+          'asyncArrow': 'always'
+        }],
+        'strict': 0,
+      },
+      'overrides': [
+        {
+          // temporarily ignoring files that violate @okta/okta/no-unlocalized-text-in-templates rule
+          // these are low priority as they are not src files
```

Comment:
To avoid another override, we should leverage the disable comment mentioned above. That way it's easier for teams to go in and fix, where they can simply search for: `eslint-disable-next-line @okta/okta/no-unlocalized-test-in-templates`.